Source/Scene/Implicit3DTileContent.js

```diff
@@ -117,7 +118,12 @@ Object.defineProperties(Implicit3DTileContent.prototype, {
 
   url: {
     get: function () {
-      return this._resource.getUrlComponent(true);
+      var resource = this._implicitTileset.subtreeUriTemplate;
+      var templateValues = this._implicitCoordinates.getTemplateValues();
+      var derivedResource = resource.getDerivedResource({
+        templateValues: templateValues,
+      });
+      return derivedResource.getUrlComponent(true);
     },
   },
```

Comment:
I'm not sure how frequently `url` would get called but could the url be computed in the constructor and stored as a member variable `this._url`, making this a simpler, faster getter?